cpp/harvest/marc_issn_lookup.cc

```diff
@@ -148,6 +148,13 @@ struct CacheEntry {
                 else
                     preferred_title_ = "";
             }
+
+            if (field.getTag() == "776") {
+                if (not field.getFirstSubfieldWithCode('x').empty()) {
+                    preferred_issn_ = StringUtil::ASCIIToUpper(field.getFirstSubfieldWithCode('x'));
```

Comment:
I would assume that most entries have a 022 field, and in addition a 776 field.
With your suggested change, preferred_issn_ will always be overridden by 776, but this should only be done if it is still empty at this point.
And also => why didn't you add the ppn + issn to related_ppns_and_its_issns? (and by the way, that struct member variable name should end with underscore_ like the others)